stereo/include/pcl/stereo/stereo_grabber.h

```diff
@@ -68,7 +68,7 @@ namespace pcl
       /** \brief Copy constructor.
         * \param[in] src the Stereo Grabber base object to copy into this
         */
-      StereoGrabberBase (const StereoGrabberBase &src) : Grabber (), impl_ ()
+      StereoGrabberBase (const StereoGrabberBase &src) : impl_ ()
```

Comment:
Same comment here. Just from this snippet this looks like a default copy ctor. Make it default?